views/layouts/main.handlebars

```diff
@@ -1 +1,15 @@
-{{{body}}}
\ No newline at end of file
+{{{body}}}
+<div class="footer">
+  <div class="legal-links">
+    <a  href="https://www.mozilla.org"><img class="mozilla-logo" src="/resources/mozilla-logo.svg"/></a>
+    <a href="https://www.mozilla.org/about/legal/">Legal</a>
+    <a href="https://testpilot.firefox.com/about">About Test Pilot</a>
+    <a href="https://testpilot.firefox.com/privacy">Privacy</a>
+    <a href="https://testpilot.firefox.com/terms">Terms</a>
+    <a href="https://www.mozilla.org/en-US/privacy/websites/#cookies">Cookies</a>
+  </div>
+  <div class="social-links">
+    <a href="https://github.com/mozilla/testpilot" target="_blank"><img class="github" src="/resources/github-icon.svg"/></a>
+    <a href="https://twitter.com/FxTestPilot" target="_blank"><img class="twitter" src="/resources/twitter-icon.svg"/></a>
+  </div>
+</div>
```

Comment:
Me thinks that this would push the footer outside of the `<html>` tag. I'll file a new bug shortly.

```html
  </div>
</div>
</body>
</html>

<div class="footer">
  <div class="legal-links">
    <a  href="https://www.mozilla.org"><img class="mozilla-logo" src="/resources/mozilla-logo.svg"/></a>
    <a href="https://www.mozilla.org/about/legal/">Legal</a>
    <a href="https://testpilot.firefox.com/about">About Test Pilot</a>
    <a href="https://testpilot.firefox.com/privacy">Privacy</a>
    <a href="https://testpilot.firefox.com/terms">Terms</a>
    <a href="https://www.mozilla.org/en-US/privacy/websites/#cookies">Cookies</a>
  </div>
  <div class="social-links">
    <a href="https://github.com/mozilla/testpilot" target="_blank"><img class="github" src="/resources/github-icon.svg"/></a>
    <a href="https://twitter.com/FxTestPilot" target="_blank"><img class="twitter" src="/resources/twitter-icon.svg"/></a>
  </div>
</div>
```